openssh/init.sls

```diff
@@ -5,10 +5,16 @@ openssh:
   pkg.installed:
     - name: {{ openssh.server }}
   {% endif %}
+  {% if openssh.sshd_enable is sameas true %}
```

Comment:
I'm not sure if/why that would be a bad thing, but I suppose it's not a huge deal either way...